todolist/settings.py

```diff
@@ -23,7 +23,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["0.0.0.0"]
```

Comment:
In order to allow everything use *
```suggestion
ALLOWED_HOSTS = ["*"]
```